guide/14-deep-learning/how_retinanet_works.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<p> </p><p>FPN creates an architecture with rich semantics at all levels as it combines low-resolution semantically strong features with high-resolution semantically weak features.</p><p>See:</p><p><a href="https://towardsdatascience.com/review-fpn-feature-pyramid-network-object-detection-262fc7482610" target="_blank">https://towardsdatascience.com/review-fpn-feature-pyramid-network-object-detection-262fc7482610</a></p><p>the lines are somewhat taken from the above link, So we should provide reference of this link also.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>These lines are taken from the paper. <a href="https://arxiv.org/pdf/1612.03144.pdf" target="_blank">https://arxiv.org/pdf/1612.03144.pdf</a> Page 2, 3rd paragraph. I have referenced the paper in the Reference section. </p><p>This blog also must have taken it from the paper.</p>